distribution/lib/Standard/Base/0.0.0-dev/src/Any.enso

```diff
@@ -83,18 +82,18 @@ type Any
         self_meta = Meta.meta self
         that_meta = Meta.meta that
         case Pair.new self_meta that_meta of
-            Pair.Value (Meta.Atom_Data _) (Meta.Atom_Data _) ->
+            Pair.Value (Meta.Atom.Atom_Data _) (Meta.Atom.Atom_Data _) ->
                 c_1 = self_meta.constructor ...
                 c_2 = that_meta.constructor ...
                 if Meta.is_same_object c_1 c_2 . not then False else
                     f_1 = self_meta.fields
                     f_2 = that_meta.fields
                     0.up_to f_1.length . all i-> (f_1.at i) == (f_2.at i)
-            Pair.Value (Meta.Error_Data _) (Meta.Error_Data _) -> self_meta.payload == that_meta.payload
-            Pair.Value (Meta.Polyglot_Data o_1) (Meta.Polyglot_Data o_2) ->
-                langs_match = (self_meta.get_language == Meta.Java) && (that_meta.get_language == Meta.Java)
+            Pair.Value (Meta.Error.Error_Data _) (Meta.Error.Error_Data _) -> self_meta.payload == that_meta.payload
+            Pair.Value (Meta.Polyglot.Polyglot_Data o_1) (Meta.Polyglot.Polyglot_Data o_2) ->
+                langs_match = (self_meta.get_language == Meta.Language.Java) && (that_meta.get_language == Meta.Language.Java)
                 if langs_match.not then False else o_1.equals o_2
-            Pair.Value (Meta.Unresolved_Symbol_Data _) (Meta.Unresolved_Symbol_Data _) ->
+            Pair.Value (Meta.Unresolved_Symbol.Unresolved_Symbol_Data _) (Meta.Unresolved_Symbol.Unresolved_Symbol_Data _) ->
```

Comment:
`Meta.Unresolved_Symbol.Unresolved_Symbol_Data` - that's very verbose and duplicated, in my opinion.